pkg/build/api/types.go

```diff
@@ -10,8 +10,12 @@ import (
 const (
 	// BuildAnnotation is an annotation that identifies a Pod as being for a Build
 	BuildAnnotation = "openshift.io/build.name"
+	// DeprecatedBuildLabel is old value of BuildLabel, it'll be removed in OpenShift 3.1.
+	DeprecatedBuildLabel = "build"
 	// BuildLabel is the key of a Pod label whose value is the Name of a Build which is run.
-	BuildLabel = "build"
+	BuildLabel = "io.openshift.build.name"
```

Comment:
@liggitt correct, but the CLI issue is still valid:

`oc delete build all -l openshift.io/build/name=foo` vs. `oc delete build all -l build=foo`